README.template.md

```diff
@@ -51,9 +51,53 @@ Here is an example to get you started:
 {{ usageExampleCode }}
 ```
 
+In addition to the `SimpleCacheClient` client library shown in the example above, a PHP PSR-16 "SimpleCache"
+implementation is also included in the SDK. See the [PSR-16 client README](README-PSR16.md) for more details.
+
 ### Error Handling
 
-Coming soon!
+Errors that occur in calls to `SimpleCacheClient` methods are surfaced to developers as part of the return values of
+the calls, as opposed to by throwing exceptions. This makes them more visible, and allows your IDE to be more
+helpful in ensuring that you've handled the ones you care about. (For more on our philosophy about this, see our
+blog post on why [Exceptions are bugs](https://www.gomomento.com/blog/exceptions-are-bugs). And send us any
+feedback you have!)
+
+The preferred way of interpreting the return values from SimpleCacheClient methods is
+using [Pattern matching](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/functional/pattern-matching).
```

Comment:
you probably don't want the .NET link here.

Reply:
Haha, wow. Yes, you are correct.